glean-core/csharp/Glean/Glean.csproj

```diff
@@ -127,4 +127,12 @@
       <_Parameter1>GleanTests</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Serilog" Version="2.9.0" />
```

Comment:
Would you kindly check if this dependency is transitively added to the sample application?

Reply:
Sure!  I was planning on trying out the suggestions from Serilog on resolving the issue I ran into with the crash so I will take it for a test drive in the sample application too.